exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/MetricAdapter.java

```diff
@@ -92,6 +92,8 @@ private static String cleanMetricName(String descriptorMetricName) {
         return Collector.Type.SUMMARY;
       case HISTOGRAM:
         return Collector.Type.HISTOGRAM;
+      case EXPONENTIAL_HISTOGRAM:
+        return Collector.Type.UNKNOWN; // todo exporter for exponential histogram
```

Comment:
There's an implied question here -- what should an exporter or processor do when it sees this data and only knows about the old style of histogram.  The same question comes up (and is even more pressing) for the OTel collector. I prototyped a converted to produce explicit-boundary histograms here: https://github.com/open-telemetry/opentelemetry-collector/pull/3841

For Prometheus, there will be special considerations -- an auto-scaling aggregator is going to create problems.

Reply:
Interesting point. Converting to explicit boundary histogram could work. I also wonder how various backends would handle that. I certainly don't have the answers for that.